contracts/src/v0.8/ccip/pools/USDC/USDCTokenPool.sol

```diff
@@ -72,6 +72,10 @@ contract USDCTokenPool is TokenPool, ITypeAndVersion {
   // A mapping of CCIP chain identifiers to destination domains
   mapping(uint64 chainSelector => Domain CCTPDomain) private s_chainToDomain;
 
+  // Mapping specifically for SVM chains, stores a fixed ATA for each chain
```

Comment:
Needs more information why this is done. Anyone should be able to understand without additional context.